chemprop/train/molecule_fingerprint.py

```diff
@@ -131,13 +131,25 @@ def molecule_fingerprint(args: FingerprintArgs, smiles: List[List[str]] = None)
 
     # Set column names
     fingerprint_columns = []
-    if len(args.checkpoint_paths) == 1:
-        for j in range(total_fp_size):
-            fingerprint_columns.append(f'fp_{j}')
-    else:
-        for j in range(total_fp_size):
-            for i in range(len(args.checkpoint_paths)):
-                fingerprint_columns.append(f'fp_{j}_model_{i}')
+    if args.fingerprint_type == 'MPN':
+        if len(args.checkpoint_paths) == 1:
+            for j in range(total_fp_size//2):
```

Comment:
What's this `//2`? Is this a hardcoding for using 2 as the args.number_of_molecules?